contracts/token/BNT.sol

```diff
@@ -0,0 +1,456 @@
+/**
```

Comment:
We don't need to build or manage this code nor VBNT.sol code here. Please only add the interfaces, the typechain, and the binary code as discussed, so that it can be referenced externally

Reply:
And in any case, we should never add any solidity code without tests (although, in this case, the code itself is redundant)

Reply:
1. The typechain is published through the npm package. In addition, we need the whole contract in order to be able to use it in migration scripts. Ideally, when we'll deploy to rinkeby, the BNT and vBNT token contracts need to be the same than on mainnet so hence the reason why I put the whole contract and not only the interfaces.

2. This code comes directly from our verified contracts on etherscan. I don't see the point of testing that.